Toolchain/BuildClang.sh

```diff
@@ -208,44 +207,13 @@ pushd "$DIR/Tarballs"
             git init > /dev/null
             git add . > /dev/null
             git commit -am "BASE" > /dev/null
+            git am "$DIR"/Patches/llvm-backport-objcopy-update-section.patch > /dev/null
```

Comment:
any reason this one is git am and the llvm.patch is git apply?

Reply:
The two patches aren't in the same format.

The objcopy patch is in the `git format-patch` format, while `llvm.patch` is a simple diff file. `git am` can't apply diffs, while `git apply` discards the commit information.